nixos/tests/guix/publish.nix

```diff
@@ -16,7 +16,7 @@ in {
       # substitute server which requires Avahi.
       services.avahi = {
         enable = true;
-        nssmdns = true;
+        nssmdns4 = true;
```

Comment:
FYI, this has already been fixed in https://github.com/NixOS/nixpkgs/pull/273491

Reply:
Got it.